ViewModel/Study/FDACache.cs

```diff
@@ -891,7 +938,10 @@ public ChildElement GetChildElementOfType(Type childElementType, int ID)
                     }
                 }
             }
-
+            else if(childElementType == typeof(AlternativeComparisonReportElement))
```

Comment:
There's another duplicate above.
These duplicates are probably caused by some of the merging stuff we were seeing before.

Reply:
removed duplicate.